com.codemouse92.timecard.yaml

```diff
@@ -0,0 +1,65 @@
+app-id: com.codemouse92.timecard
+default-branch: stable
+base: io.qt.qtwebkit.BaseApp
+base-version: "5.14"
+runtime: org.kde.Platform
+runtime-version: "5.14"
+sdk: org.kde.Sdk
+command: Timecard-App
+cleanup:
+    - /include
+    - /share/aclocal
+    - /share/doc
+    - /share/cmake
+    - /share/man
+    - "*.a"
+    - "*.la"
+finish-args:
+    - --filesystem=host
+    - --share=ipc
+    - --socket=x11
+    - --socket=fallback-x11
+    - --socket=wayland
+    - --device=dri
+    - --talk-name=org.freedesktop.DBus.Proprieties
+    - --talk-name=org.freedesktop.IBus
+    - --talk-name=org.freedesktop.Notifications
+    - --talk-name=org.kde.StatusNotifierWatcher
+    - --own-name=org.kde.*
+    - --talk-name=com.canonical.indicator.application
+    - --talk-name=org.ayatana.indicator.application
+    - --env=PATH=/app/bin
+    - --env=PYTHONPATH=/app/packages/python
+modules:
+    - name: pyside2
+      buildsystem: cmake-ninja
+      builddir: true
+      config-opts:
+          - -DCMAKE_BUILD_TYPE=Release
+          - -DBUILD_TESTS=OFF
+      cleanup:
+          - /bin
+      sources:
+        - type: archive
+          url: https://download.qt.io/official_releases/QtForPython/pyside2/PySide2-5.15.0-src/pyside-setup-opensource-src-5.15.0.tar.xz
+          sha256: f1cdee53de3b76e22c1117a014a91ed95ac16e4760776f4f12dc38cd5a7b6b68
+        - type: shell
+          commands:
+           - mkdir -p /app/include/qt5tmp && cp -R /usr/include/Qt* /app/include/qt5tmp  # https://bugreports.qt.io/browse/PYSIDE-787
+           - sed -i 's|\(--include-paths=\)|\1/app/include/qt5tmp:|' sources/pyside2/cmake/Macros/PySideModules.cmake
+
+    - name: Timecard-App
+      buildsystem: simple
+      build-options:
+        build-args:
+          - "--share=network"
+      build-commands:
+        - python3 setup.py install --prefix=${FLATPAK_DEST}
+        - install -Dm644 app/share/icons/${FLATPAK_ID}.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg
+        - install -Dm644 app/share/applications/${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+        - install -Dm644 app/share/metainfo/${FLATPAK_ID}.appdata.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.appdata.xml
+      sources:
+        - type: git
+          url: https://github.com/codemouse92/timecard
+          branch: stable
+
```

Comment:
```suggestion
    branch: v2.0.5
    commit: bc0dc8e4690d83eaef15abee50875ed49fd35a28
```

Reply:
Just to clarify: you need a specific commit. or a tag. (or both)

A bare branch isn't enough.